src/pages/login/login.ts

```diff
@@ -39,6 +46,10 @@ export class LoginPage extends BasePageComponent {
     }
   }
 
+  ionViewDidEnter(): void {
+    this.analytics.setCurrentPage(AnalyticsScreenNames.LOGIN);
```

Comment:
How does this behave given you won't be logged in at the time of seeing this page?

Reply:
It uses the UDID for logging (which will change to a hashed UDID in the future). So it can log that this page is accessed (so long as connected to internet).